DM-27414: update for SAL component Rotator -> MTRotator

r-owen opened the pull request.

wvreeven commented on bin/command_rotator.py:
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
```
In the ts_mthexapod project you renamed the run_hexapod.py and command_hexapod.py scripts but you do not rename the command_rotator.py script. I suggest you do to have a homogeneous naming of your scripts. The same applies to the run_rotator.py script, which I see you did not rename either. Don't forget to update the files in the project that refer to these scripts.

r-owen replied:
Good catch. I'll make these changes. Thanks!

wvreeven commented on doc/version_history.rst:
```diff
@@ -1,11 +1,32 @@
+* ts_simactuators 1
+* ts_idl 2.2
+* ts_xml 7
+* Rotator IDL files, e.g. made using ``make_idl_files.py Rotator``
```
Please fix this to refer to MTRotator twice.

wvreeven commented on tests/test_csc.py:
```diff
@@ -45,7 +45,7 @@ async def test_bin_script(self):
-            name="Rotator",
+            name="MTRotator",
             index=None,
             exe_name="run_rotator.py",
```
Don't forget to update this line when you rename the run_rotator.py script to run_mtrotator.py.

wvreeven reviewed (approved):
I only found one important change to do: rename the scripts in the bin directory. Otherwise this looks good to me.

r-owen commented:
Thank you for catching those issues. I made the following changes:
* Updated Rotator -> MTRotator for the IDL instructions in the most recent entry in the version history.
* Renamed all bin scripts to change "_rotator" to "_mtrotator".
* Searched for _rotator and changed all instance to _mtrotator. This caught most of the issues you mentioned, including the bin scripts listed in setup.py.

I also looked for _hexapod in ts_hexapod and found and fixed a few that I had missed the first time.
